releases/release-1.31/release-notes/maps/pr-126108-map.yaml

```
@@ -0,0 +1,7 @@
+pr: 126108
+releasenote:
+  text: |-
+    Reduced state change noise when volume expansion fails. Also mark certain failures as infeasible.
+
+    If you are using the RecoverVolumeExpansionFailure Alpha feature, after upgrading to this release, existing PVCs with status.allocatedResourceStatus set to "ControllerResizeFailed" or "NodeResizeFailed" should have their status.allocatedResourceStatus cleared.
```

```suggestion
    If you are using the `RecoverVolumeExpansionFailure` alpha feature, after upgrading to this release, existing PVCs with `status.allocatedResourceStatus` set to ControllerResizeFailed or NodeResizeFailed should have their `status.allocatedResourceStatus` cleared.
```

The advice is unclear; does it say that the `status.allocatedResourceStatus` cleared should be cleared [by zombies](https://www.archives.gov/open/plain-writing/tips/passive-voice.html) so no action is needed, or that the  user (cluster adminstrator) is expected to make the change themselves?

---

I'd keep the backticks. How about:
```suggestion
 ACTION REQUIRED:  If you are using the `RecoverVolumeExpansionFailure` alpha feature gate
 then after upgrading to this release, you need to update some objects.
 For any existing PersistentVolumeClaimss with `status.allocatedResourceStatus` set to either
 "ControllerResizeFailed" or "NodeResizeFailed", clear the `status.allocatedResourceStatus`.
```